maintainers/scripts/pluginupdate.py

```diff
@@ -514,6 +543,14 @@ def update_plugins(editor: Editor):
             [args.outfile, args.input_file, editor.deprecated],
         )
 
+    if args.search:
+        results = search_github(editor)
+        with open(args.input_file, "r") as f:
+            lines = {line.strip("\n") for line in f.readlines()}
+            for plugin in results.intersection(lines):
+                results.remove(plugin)
+            args.add_plugins.extend(results)
```

Comment:
This can be `arg.add_plugins.extend(results - lines)`.
Also can be outside the `with` block.